tests/chainer_tests/functions_tests/connection_tests/test_convolution_2d.py

```diff
@@ -137,10 +148,13 @@ def forward(self, inputs, device):
             b = None
         else:
             x, W, b = inputs
-        return F.convolution_2d(
+        out = F.convolution_2d(
             x, W, b, stride=self.stride, pad=self.pad,
             cover_all=self.cover_all, dilate=self.dilate,
-            groups=self.groups),
+            groups=self.groups)
+        if self.old_numpy_fp16:
+            return out*0,
```

Comment:
```suggestion
            return out*0,  # See #8458
```

Reply:
Its explained in the setUp comment 😄

https://github.com/chainer/chainer/pull/8458/files#diff-0c14a0d1b376c42cedd98bb914521e8bR80-R83

Reply:
ditto.